spec/clipboard.spec.js

```diff
@@ -144,11 +144,16 @@ describe('Clipboard', () => {
       expect(parseContent(div)[0]).toEqual('bar')
     })
 
-    it('keeps internal links of a anchorTag with an href attribute', () => {
-      expect(extractSingleBlock('<a href="http://localhost:9876/test123">a</a>')).toEqual('<a href="http://localhost:9876/test123">a</a>')
+    it('changes absolute links to relative ones with the keepInternalRelativeLinks flag set to true', () => {
       config.pastedHtmlRules.keepInternalRelativeLinks = true
       updateConfig(config)
```

Comment:
Each test should restore the config to the state before. Or is that already done somewhere?